### PR TITLE
Mon 72016 initialize the test for alma9

### PR DESCRIPTION
**Pull request by sfarouq-ext**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by tuntoja — COMMENTED** [review]
Please use the following pull request title 
`enh(tests): Initialize gorgone test environement on EL9`

File: .github/docker/Dockerfile.gorgone-testing-alma9 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM almalinux:9
+
+RUN bash -e <<EOF
+
+dnf install -y dnf-plugins-core zstd mariadb iproute
+dnf install -y epel-release 
+dnf config-manager --set-enabled crb
+dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/23.10/el9/centreon-23.10.repo
+dnf -y clean all --enablerepo=*
```
Comment (Evan-Adam) [inline .github/docker/Dockerfile.gorgone-testing-alma9:9]: This line seem uselless as you install other depedancy on the next line, and already clean dnf on line 13

Comment (sfarouq-ext) [reply .github/docker/Dockerfile.gorgone-testing-alma9:9]: i removed the line 13 and kept the line 9

Comment (Evan-Adam) [reply .github/docker/Dockerfile.gorgone-testing-alma9:9]: You can't do that as you install again package line 10

**Comment by kduret** [comment]
Migrated to https://github.com/centreon/centreon-collect/pull/1526

**Pull request closed without merging**